### PR TITLE
Feature/split gallery

### PR DESCRIPTION
**Pull request by dreitzner**
- rework gallery into it's own route
- clean up a bit 🧹
- package updates
- move to new eslint for svelte
- more opinionated prettier
- guard gallery from wrong entries on server (int check, max picture check)
- format all files

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -8,13 +8,10 @@
 		"resolveJsonModule": true,
 		"skipLibCheck": true,
 		"sourceMap": true,
-		"strict": true,
-		"noImplicitAny": true,
```
Comment (ivanhofer) [inline tsconfig.json:12]: Why did you remove this strict options?

**Review by ivanhofer — APPROVED** [review]
Thanks for updating the website!
The build is failing!

**Comment by dreitzner** [comment]
> Thanks for updating the website! The build is failing!

@ivanhofer we no longer deploy to cloudflare, as we are allowed to use the generous vercel offer.

File: tsconfig.json (hunk 1 above)
Comment (dreitzner) [reply tsconfig.json:12]: I synced some of the configs with the create svelte package.

In this case: https://github.com/sveltejs/kit/blob/master/packages/create-svelte/shared/%2Btypescript/tsconfig.json

**Comment by ivanhofer** [comment]
> we no longer deploy to cloudflare, as we are allowed to use the generous vercel offer.

Then we should get rid of the integration ^^

File: tsconfig.json (hunk 1 above)
Comment (ivanhofer) [reply tsconfig.json:12]: The options you have removed make TypeScript less strict

**Comment by dreitzner** [comment]
> > we no longer deploy to cloudflare, as we are allowed to use the generous vercel offer.
> 
> Then we should get rid of the integration ^^

Kev already did that for us (after I noticed it in the PR) 😉

File: tsconfig.json (hunk 1 above)
Comment (dreitzner) [reply tsconfig.json]: Ok, I added them back :)
